src/main/java/onlyoffice/FileUtil.java

```diff
@@ -0,0 +1,153 @@
+/**
+ *
+ * (c) Copyright Ascensio System SIA 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package onlyoffice;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+
+import java.util.Collection;
+import java.util.Arrays;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import com.atlassian.jira.issue.attachment.Attachment;
+
+@Named
+public class FileUtil {
```

Comment:
you can immediately add a formats matrix as done in nextcloud (https://github.com/ONLYOFFICE/onlyoffice-nextcloud/blob/master/lib/appconfig.php#L1099-L1134)